README.md

```diff
@@ -221,9 +214,10 @@ pancake --verbose
 
 💡 You can use Pancake with your own modules. All you have to do in your modules is:
 
-1. add the Pancake module object to your `package.json` file
-2. add the Pancake `postinstall` script and dependency to your `package.json` file
-3. and add your peer dependencies
+1. Install Pancake with `npm i @gov.au/pancake`.
```

Comment:
Maybe specify that installing pancake isn't necessary if you install a design system component as they have it built in.

Reply:
The numbering doesn't line up with the detailed section after this.

Reply:
I hope that the get started section would help users know they do not need to be in the "Creating your own Pancake modules" section.

I have updated the titles.